pkg/controller/podWatcher.go

```diff
@@ -163,7 +163,7 @@ func podEventsHandler(key string, indexer cache.Indexer) error {
 		}
 
 		if podAnnotations != nil {
-			ignoreEvent = podAnnotations[ignorePodUpdateAnnotationName] == "true"
+			watchEvent = podAnnotations[ignorePodUpdateAnnotationName] == "true"
```

Comment:
`ignorePodUpdateAnnotationName` variable is not exists anymore. you change he's name (line 20) to `watchPodUpdateAnnotationName`

Reply:
fixed